internal/learn/the-community/discord-verification.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Discord Verification
```

Comment:
Was there a specific reason to rename the title and filename? If not, let's just keep it short. It's about Discord, so lets just call it `discord`.

Reply:
Yes, this page is now specific to the Discord verification procedure which is an important topic.
Since it's the first time doing this, and the opportunity that Discord will have more specific pages, it would help me a lot to get suggestions about how to make this better to separate specific information based on the platform